analysis_options.yaml

```diff
@@ -19,6 +19,9 @@ analyzer:
     # Ignore generated files
     - '**/*.g.dart'
     - '**/*.mocks.dart' # Mockito @GenerateMocks
+    # Ignore go_router dart fix test file.
```

Comment:
I am not sure if this is the right place to disable the lint

Reply:
I think you could make a custom analysis_options.yaml in `packages/go_router/` that includes this one, and then adds this exclude. Then you'd need to add go_router to `script/configs/custom_analysis.yaml`.

That would make it more obvious locally that something strange is going on with analysis.

Reply:
hmm, this doesn't work. In order to let dart fix --compare-to-golden to work. I can't ignore the lint using analysis_options.yaml as the dart fix would only touch the code that has analyzer error or warning. That means I can't do any kind or ignore that would affect `dart analyze`.

I took a look at the flutter repro, it looks like they wrote the custom analyzer script to ignore these lint.

I think I will have to need to update the code in flutter_plugin_tools to ignore the folder when running analyze.

Do you have any other idea that I should do?

Reply:
nvm, it seems like i can use some analyzer-options trick to get around this